dask_image/utils/_dispatcher.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import scipy.ndimage.filters
+from dask.utils import Dispatch
+
+
+class Dispatcher(Dispatch):
+    """Simple single dispatch for different dask array types."""
+
+    def __call__(self, arg, *args, **kwargs):
+        """
+        Call the corresponding method based on type of dask array.
+        """
+        meth = self.dispatch(type(arg._meta))
+        return meth(arg, *args, **kwargs)
```

Comment:
I think with the changes below we can drop this.

Reply:
I'm not sure I understand what you mean by dropping this.
The important part here is that we use `type(arg._meta)` to dispatch according to the type of arrays contained in the dask array. So that part matters, but other than that I don't mind what we do.